### ISSUE TITLE
Coherence is broken with teh latest Twisted version (16.0.0)

### ISSUE THREAD

**Issue by wil93** [coherence-project/Coherence#25]
The latest stable (0.6.6.2) checks:

```
if twisted_web_version < Version("twisted.web", 2, 5, 0):
```

But with twisted/twisted@9bc27f66e6fe30dc1aa248d8b684f24df83401ac the version field was changed for all submodules, and the `.web` part was dropped.

Maybe the check should be changed so that, when it throws an `twisted.python.versions.IncomparableVersions` exception, it checks:

```
if twisted_web_version < Version("twisted", 16, 0, 0):
```

**Comment by diederikh** [comment]
I have the same issue on OS X 10.11, Python 2.7.10

**Comment by hawkowl** [comment]
(hi, I broke your code!)

It should not check against the twisted web version, only the Twisted version (`twisted.__version__`) in all cases, as that's part of Core, which was always available even pre-subproject removal, and Twisted and Twisted Web's version must have always been the same previously anyway.

**Comment by htgoebel** [comment]
This project is seeking a maintainer, see #33. If you want this to be fixed, please provide a pull-request or volunteer to maintain the project.

**Comment by wil93** [comment]
I think this was already fixed by PR #27
